tests/fixtures/models/model-relationships-morphone-morphmany-laravel8.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Flag extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'user_id',
+    ];
+
+    /**
+     * The attributes that should be cast to native types.
+     *
+     * @var array
+     */
+    protected $casts = [
+        'id' => 'integer',
+        'user_id' => 'integer',
+    ];
+
+
+    public function order()
+    {
+        return $this->morphOne(\Other\Package\Order::class, 'starable');
+    }
+
+    public function stars()
+    {
+        return $this->morphOne(\App\Some::class, 'starable');
+    }
+
+    public function durations()
+    {
+        return $this->morphMany(\Other\Package\Duration::class, 'durationable');
+    }
+
+    public function transactions()
```

Comment:
Shouldn't this method name be `lines`?

Reply:
Solved as in last comment! 

Thanks again 👍